src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1293,6 +1293,11 @@ public override BoundNode VisitLiteral(BoundLiteral node)
             return null;
         }
 
+        public override BoundNode VisitMethodToken(BoundMethodToken node)
+        {
+            return null;
```

Comment:
My thoughts on this:
- Instrumentation must be a separate pass. This way instrumentation would not impact performance in regular scenarios.
- Instrumentation should use unlowered tree since it is isomorphic to the code